charts/kong/values.yaml

```
@@ -439,17 +439,19 @@ migrations:
 dblessConfig:
   # Either Kong's configuration is managed from an existing ConfigMap (with Key: kong.yml)
   configMap: ""
+  # Or Kong's configuration is managed from an existing Secret (with Key: kong.yml)
+  secret: ""
   # Or the configuration is passed in full-text below
-  config: |
-    _format_version: "1.1"
-    services:
-      # Example configuration
-      # - name: example.com
-      #   url: http://example.com
-      #   routes:
-      #   - name: example
-      #     paths:
-      #     - "/example"
+  config: "" # |
```

`ct lint` is being stupid about comments:

```
charts/kong/values.yaml
  445:14    error    too few spaces before comment  (comments)
```
It looks like the original actual `|` was fine since it's just whitespace uncommented after, so it defaults to empty string even though it's multi-line.

```suggestion
  config: |
```

---

after some testing, it turns out YAML doesn't parse leading `#`s as comment markers if they're inside a block scalar, they're treated as literal text. so the resulting string is not empty, despite looking empty. normally this would be fine, but now that we have a rule enforcing the mutual exclusivity of these fields, `config` having a non-empty string value by default is a problem. to get around this, we need the example content comments to be indented at the same level as the `config` key. Here's the best compromise I could think of:
```
  config: |
  # # _format_version: "1.1"
  # # services:
...
```
(the second # character is optional, it could be replaced with a space based on aesthetic preference.)

---

Weird, how was it not breaking before?

Eh, whatever, if the double comments work, looks like it's just now git being boneheaded about conflict detection in the changelog.

---

oh, it was working before because there was no logic to explicitly fail if both values were non-empty, that was added in this PR as well. I'll rebase to fix the changelog and then I think that's everything!